lib/bgs/services/benefit_claims.rb

```diff
@@ -30,5 +30,22 @@ def find_bnft_claim(claim_id:)
       response = request(:find_bnft_claim, bnftClaimId: claim_id)
       response.body[:find_bnft_claim_response]
     end
+
+    # Note in BGS docs: A call to findBnftClaim should be made prior to calling this method,
+    # as each value returned in findBnftClaim should be passed as input to updateBnftClaim, along with any updates.
+    # Failure to call findBnftClaim and provide all the date runs a risk of data corruption.
+    def update_bnft_claim(claim:)
+      # This works in rails, but not pure ruby
+      # claim.deep_transform_keys! { |key| key.to_s.camelize(:lower).to_sym }
+      # This isn't as pretty, but works in pure ruby
+      claim.transform_keys! do |key|
+        str = key.to_s.downcase.split('_').collect(&:capitalize).join
+        str[0] = str[0].downcase
+        str.to_sym
+      end
+
+      response = request(:update_bnft_claim, { bnftClaimDTO: claim }, claim[:bnftClaimId])
```

Comment:
doesn't the find give us a response with 
```
{:bnft_claim_dto=>{:bnft_claim_id=>
```
looking at the PR for vets-api, it looks like the claim comes back with the above.
if that's the case we wouldn't necessarily need `bnftClaimDTO` in 47, but we could actually keep that and just do `claim[:bnft_claim_dto].transform_keys!` in 41 and pass that to 47 as the claim so we don't have to do some weird double nesting with `transform_keys!`